helper/config/config.go

```diff
@@ -0,0 +1,28 @@
+package config
```

Comment:
As long as we're making a new package, maybe it would be nice if this were something we could use for other IP-address-related utilities? What if we called it `helper/ipaddress` instead?